provide pt.json translate

celosxe opened the pull request:
Translate PtBr

stefanseifert commented:
thanks for the contribution, this look good!
i fixes two small issues with placeholders (placeholder names in brackets must not be translated).
i also renamed the file to "pt-br" as the translation is brazilian portuguese.

celosxe commented:
Tnx man!

"Etre et durer... Etre fort pour être utille"

Em seg., 28 de jul. de 2025, 07:19, Stefan Seifert ***@***.***>
escreveu:

> *stefanseifert* left a comment (brdgm/ark-nova-solo-helper#53)
> <https://github.com/brdgm/ark-nova-solo-helper/pull/53#issuecomment-3126548997>
>
> thanks for the contribution, this look good!
> i fixed two small issues with placeholders (placeholder names in brackets
> must not be translated).
> i also renamed the file to "pt-br" as the translation is brazilian
> portuguese.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/brdgm/ark-nova-solo-helper/pull/53#issuecomment-3126548997>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BEPQRSSJQN6NOASVPA2QO4L3KX2J5AVCNFSM6AAAAACCQKEFK6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTCMRWGU2DQOJZG4>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>
